### PR TITLE
Add Debian package

### PR DESCRIPTION
**Pull request by NicolasLM**

### CODE REVIEW

File: debian/sauna.postinst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+create_user () {
+
+    adduser --system --quiet --group --no-create-home sauna || true
+
+}
+
+create_base_config_file () {
+
+    if [ ! -f /etc/sauna.yml ]; then
```
Comment (bewiwi) [inline debian/sauna.postinst]: if [ ! -e ....

This can be avoid to erase user symlink

File: debian/sauna.postrm [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+set -e
+
+
+delete_base_config_file () {
+
+    if [ -f /etc/sauna.yml ]; then
+        rm /etc/sauna.yml
```
Comment (bewiwi) [inline debian/sauna.postrm:8]: As a user i don't want loose my personal config file on package remove

Comment (NicolasLM) [inline debian/sauna.postrm:8]: This is only executed for purge, hence removing the configuration file.

File: debian/sauna.postinst (hunk 1 above)
Comment (NicolasLM) [inline debian/sauna.postinst]: done

**Pull request merged**